INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 328-795-710
ERROR demo2.py:6 error info
WARNING demo2.py:7 warn info
INFO demo2.py:8 info info
DEBUG demo2.py:9 debug info
INFO _internal.py:87 127.0.0.1 - - [28/Jan/2018 15:45:26] "GET /api/v1.0/index HTTP/1.1" 200 -
ERROR demo2.py:6 error info
WARNING demo2.py:7 warn info
INFO demo2.py:8 info info
DEBUG demo2.py:9 debug info
INFO _internal.py:87 127.0.0.1 - - [28/Jan/2018 15:45:27] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 328-795-710
ERROR demo2.py:6 error info
WARNING demo2.py:7 warn info
INFO demo2.py:8 info info
DEBUG demo2.py:9 debug info
INFO _internal.py:87 127.0.0.1 - - [28/Jan/2018 15:47:17] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/demo2.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 328-795-710
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 328-795-710
ERROR demo2.py:22 error info
WARNING demo2.py:23 warn info
INFO demo2.py:24 info info
DEBUG demo2.py:25 debug info
INFO _internal.py:87 127.0.0.1 - - [28/Jan/2018 15:48:56] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/demo2.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 328-795-710
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/demo2.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 328-795-710
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 14:51:27] "GET /verify/555 HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 14:58:49] "GET /verify/555 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 14:58:57] "GET /verify/555 HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:05:16] "GET /api/v1.0/verify/ddd HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:05:16] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:05:16] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:05:16] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:05:17] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:05:18] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:19] "GET /api/v1.0/verify/ddd HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:19] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:19] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:19] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:19] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:19] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:21] "GET /api/v1.0/verify/ddd HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:21] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:21] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:21] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:21] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:08:21] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:10:39] "GET /api/v1.0/verify/ddd HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:10:39] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:10:39] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:10:39] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:10:39] "GET /api/v1.0/verify/ddd?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 15:11:10] "GET /api/v1.0/verify/ddd HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:24:20] "GET /api/v1.0/verify/ddd HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:25:01] "GET /api/v1.0/img_code/44 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:25:04] "GET /api/v1.0/img_code/44 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:25:20] "GET /api/v1.0/img_codes/44 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:26:12] "GET /api/v1.0/img_codes/44 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:26:24] "GET /api/v1.0/img_codes/4 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:27:03] "GET /api/v1.0/img_codes/4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:27:04] "GET /api/v1.0/img_codes/4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:27:07] "GET /api/v1.0/img_codes/4 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:27:17] "GET /api/v1.0/img_codes/4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:30:47] "GET /api/v1.0/img_codes/4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:30:52] "GET /api/v1.0/img_codes/6 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 21:30:56] "GET /api/v1.0/img_codes/7 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:40] "GET / HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:42] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:43] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:44] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:52] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:52] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:52] "GET /static/css/ihome/register.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:53] "GET /api/v1.0/img_codes/fa6c2073-9a35-4c32-9543-ff9d2fb25ab4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:57] "GET /api/v1.0/img_codes/afe65974-c7ac-430f-95d9-cc42459833c7 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:58] "GET /api/v1.0/img_codes/35faecc3-e8ff-4508-ba5b-0ce903033dfb HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:59] "GET /api/v1.0/img_codes/3e151eb3-d512-4f78-87dc-4d8d40073803 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:28:59] "GET /api/v1.0/img_codes/eb63c297-44c8-4275-86db-a323ec5982ea HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:00] "GET /api/v1.0/img_codes/cb427b57-33cd-4381-bf5b-3144eef5fb24 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:01] "GET /api/v1.0/img_codes/2a5fa29a-df3b-4603-923c-4e5bb59ec56e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:01] "GET /api/v1.0/img_codes/f8d408f9-4c9c-41b9-9c95-bcd88ac7ef5c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:01] "GET /api/v1.0/img_codes/7432264d-ab67-421a-ae82-2f35feabbd81 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:01] "GET /api/v1.0/img_codes/3f9bd4c2-f980-423d-ae91-c99962fadf08 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:01] "GET /api/v1.0/img_codes/8fcb8fc9-0bd9-4ac2-a240-224c0ce06fa7 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:02] "GET /api/v1.0/img_codes/341fefe2-2a7f-41df-93ac-35b4b7f0cd0a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:02] "GET /api/v1.0/img_codes/cd474e79-3504-40b7-9eed-700fc700a05c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:02] "GET /api/v1.0/img_codes/a2d9f747-8364-403f-96eb-9e037df70d73 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:02] "GET /api/v1.0/img_codes/ffb897dd-0bb2-4d34-9ed2-cb67fb9dd50e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:02] "GET /api/v1.0/img_codes/23f30eda-7d97-4f0e-8a28-08273c4a99f3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:29:02] "GET /api/v1.0/img_codes/b9ffab35-cfdc-48f4-9c5a-249eaf535671 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:31:02] "GET /api/v1.0/img_codes/13b6516e-42e5-4766-870d-0566841ff48d HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:37:55] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:37:56] "GET /api/v1.0/img_codes/374a0006-bdc3-4427-aa67-09429bdfafc2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:19] "GET /register.htm HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:23] "GET /api/v1.0/img_codes/1ff808fb-ab76-4859-ad2c-aeb0ec144835 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:24] "GET /api/v1.0/img_codes/62eb66c5-ac7a-4b3a-aa01-18e2279d60ac HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:25] "GET /api/v1.0/img_codes/1446a3cd-6218-413a-994c-15daf3b0609e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:25] "GET /api/v1.0/img_codes/e405bc1c-1fec-4db4-affe-abacb6244b7d HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:25] "GET /api/v1.0/img_codes/27820244-c8e4-4209-bb4b-7ac83d96978a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:25] "GET /api/v1.0/img_codes/9d073e03-c69a-418e-9823-4e2230868425 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:25] "GET /api/v1.0/img_codes/b2f5a0ef-a7c5-4155-9010-3c105f12b3ad HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:26] "GET /api/v1.0/img_codes/c574b905-04df-46cf-9b71-d6bbf93e630b HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:26] "GET /api/v1.0/img_codes/28638505-a160-409f-9ccf-7cc83abd950c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:26] "GET /api/v1.0/img_codes/7f4eaccb-00c0-4138-a555-475fff007c2c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:26] "GET /api/v1.0/img_codes/f19c4583-5a48-43bb-8b4b-75b03c45797c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:26] "GET /api/v1.0/img_codes/e4518000-a8d8-4343-bcb3-d06f6bdbb4f1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:27] "GET /api/v1.0/img_codes/65fbb2fd-e776-4f1f-9502-42983bc88205 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:27] "GET /api/v1.0/img_codes/192e4cf1-d7db-4063-b66b-7cebb9196163 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:27] "GET /api/v1.0/img_codes/1a2ae40e-ad2d-4d17-8bd7-a5dab59bafa2 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:53] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:38:53] "GET /api/v1.0/img_codes/ed6e175e-4a06-4e16-9b05-1978b5d8b453 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:39:01] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:05] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:06] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:10] "GET /api/v1.0/img_codes/dfb375d1-df80-4d71-9af1-562ff2e962cb HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:12] "GET /api/v1.0/img_codes/22aa19b1-16c3-448b-beb8-8b5f0154a283 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:12] "GET /api/v1.0/img_codes/943ca593-3e9b-4e6f-bab0-2b161f10a4e0 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:13] "GET /api/v1.0/img_codes/c726f84b-4dfb-42de-b5b7-362ee49a07bc HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:13] "GET /api/v1.0/img_codes/9efd4a2d-ad74-47c9-a778-5aa47158bdcf HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:13] "GET /api/v1.0/img_codes/3c5f6324-8959-4d71-8325-77bc8da37d10 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:15] "GET /api/v1.0/img_codes/e8d70337-c85a-48e9-a2ee-2c8393eaca09 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:16] "GET /api/v1.0/img_codes/c85d04cc-1bf6-42cd-8ac1-cc112500694b HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:16] "GET /api/v1.0/img_codes/4105333b-a090-42dd-b32e-b8236338630c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:27] "GET /api/v1.0/image_codes/128 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:30] "GET /api/v1.0/image_codes/128 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:32] "GET /api/v1.0/image_codes/128 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:33] "GET /api/v1.0/image_codes/128 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:37] "GET /api/v1.0/image_codes/12 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:40:52] "GET /api/v1.0/image_codes/123 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:41:15] "GET /api/v1.0/img_codes/123 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:41:57] "GET /api/v1.0/img_codes/123 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET / HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:13] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:16] "GET /register HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:18] "GET /register HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:29] "GET /api/v1.0/img_codes/176cd461-3890-4275-a874-19e206e5fd78 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:29] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:31] "GET /api/v1.0/img_codes/01788308-8b83-4f56-84a1-0e6f4037799f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:32] "GET /api/v1.0/img_codes/986d8d36-b01d-4048-9157-114e2db05ef3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:56] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:42:57] "GET /api/v1.0/img_codes/a676201b-cbca-4f9b-8732-9613d11c8f7f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:43:03] "GET /api/v1.0/img_codes/a676201b-cbca-4f9b-8732-9613d11c8f7f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:43:28] "GET /api/v1.0/img_codes/a676201b-cbca-4f9b-8732-9613d11c8f7f HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:43:55] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:43:56] "GET /api/v1.0/img_codes/0f926fbe-a9c2-478e-8f4b-aa978c6a06a0 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:43:59] "GET /api/v1.0/img_codes/0f926fbe-a9c2-478e-8f4b-aa978c6a06a0 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:44:57] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:44:57] "GET /api/v1.0/img_codes/5e24cc34-04e9-4692-adc9-98890530c4d1 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:09] "GET /111.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:09] "GET /api/v1.0/img_codes/111/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:15] "GET /api/v1.0/img_codes/111/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:23] "GET /111.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:23] "GET /api/v1.0/img_codes/111/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:44] "GET /111.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:47:44] "GET /api/v1.0/img_codes/111/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:03] "GET /api/v1.0/img_codes/111/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:08] "GET /api/v1.0/img_codes/111/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:13] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:24] "GET /111.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:24] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:29] "GET /111.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:29] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:31] "GET /111.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:31] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:40] "GET /111.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:49:40] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:51:28] "GET /111.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:51:28] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:51:39] "GET /111.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:51:39] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:51:43] "GET /api/v1.0/img_codes/6bae8a9e-95c9-4f3b-948a-073e068823c8 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [30/Jan/2018 22:51:54] "GET /api/v1.0/img_codes/111 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [31/Jan/2018 09:58:44] "GET /api/v1.0/img_codes/1b86a001-2298-4ae5-b072-7bf1469fc32d HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [31/Jan/2018 09:58:45] "GET /register.htm HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [31/Jan/2018 09:58:46] "GET /api/v1.0/img_codes/bd0d7001-091a-4fd5-a9f1-164a8cc02092 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [31/Jan/2018 09:58:48] "GET /api/v1.0/img_codes/93180042-8f10-45e2-a581-9549b0fe6949 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [31/Jan/2018 09:58:50] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [31/Jan/2018 09:58:50] "GET /api/v1.0/img_codes/df16a212-2a60-4f11-a127-a42e9f88e5e6 HTTP/1.1" 200 -
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:34] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:35] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:36] "GET /api/v1.0/img_codes/c69f0024-5760-43d6-a542-785ece1232f6 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:34:36] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:35:13] "GET /api/smscode?mobile=18979907572&code=fzru&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:11] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:11] "GET /api/v1.0/img_codes/a28f523b-30d5-4ecb-b434-34b044789284 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:11] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:42] "GET /api/smscode?mobile=222&code=llll&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:50] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:50] "GET /api/v1.0/img_codes/396b78d9-2aff-455d-9f5b-e7643a4c3e03 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:50] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:40:59] "GET /api/smscode?mobile=1&code=uu&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:49:53] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:49:53] "GET /api/v1.0/img_codes/4695c613-be85-4ec5-b549-2852be0386bb HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:49:53] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:53:29] "GET /api/smscode?mobile=18979907572&code=kkkk&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:53:33] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 13:53:36] "GET /api/smscode?mobile=18979907572&code=kkkk&codeId= HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:49:43] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:49:46] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:49:48] "GET /api/v1.0/img_codes/ebb6d72c-f2f8-4b7c-809d-aa1fcd227dd4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:49:48] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:55:46] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:55:46] "GET /api/v1.0/img_codes/4c22db9a-1c63-41f7-9d70-fa3f0cbd0e7b HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:55:46] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:56:10] "GET /api/smscode?mobile=18979907572&code=xwnl&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 22:57:28] "GET /api/smscode?mobile=18979907572&code=xwnl&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:14] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:14] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:14] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /api/v1.0/img_codes/f8fcc2ac-598d-42e3-b7db-8cecb5418a61 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:15] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:19] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:36] "GET /api/smscode/18979907572?code=hjhjhj&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:00:43] "GET /api/smscode/18979907572?code=hjhjhj&codeId= HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:11:33] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:11:33] "GET /api/v1.0/img_codes/4e0ac1ab-cbcc-4e8e-b7fd-a1c7e3359bc1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:11:43] "GET /api/smscode/18979907572?code=hjj&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:11:48] "GET /api/smscode/18979907572?code=hjj&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:39] "GET /api/smscode/18979907572?code=hjhjhj&codeId= HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:46] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:46] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:47] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:48] "GET /api/v1.0/img_codes/a562a065-10bc-4fc0-8683-ff728981edc3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:12:57] "GET /smscode/18979907572?code=fghjk&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:13:11] "GET /api/v1.0/img_codes/cd66eaa2-45fc-47ae-9075-0e247932134f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:13:20] "GET /smscode/18979907572?code=uymg&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:13:31] "GET /smscode/18979907572?code=uymg&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:15:42] "GET /smscode/18979907572?code=uymg&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:16:42] "GET /smscode/18979907572?code=uymg&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:17:41] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:17:41] "GET /api/v1.0/img_codes/2aeab07a-9f17-4868-9b3e-a439eb374614 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:17:55] "GET /smscode/18979907572?code=ukwo&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [12/Feb/2018 23:18:00] "GET /smscode/18979907572?code=ukwo&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:23:43] "GET /api/v1.0/img_codes/d26d31e1-e460-4906-b019-40c084ad56c4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:23:45] "GET /api/v1.0/img_codes/f66b62d5-517d-40a7-86ec-d471f8d5aa38 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:23:46] "GET /api/v1.0/img_codes/f66b62d5-517d-40a7-86ec-d471f8d5aa38 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:24:21] "GET /api/v1.0/img_codes/456a87e5-c8c6-4264-9042-184c127840b0 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:24:22] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:24:23] "GET /api/v1.0/img_codes/40a92cc5-24fe-4b8e-98e5-b364f5d88df3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:24:46] "GET /smscode/18979907572?code=00&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:24:48] "GET /smscode/18979907572?code=00&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:24:49] "GET /smscode/18979907572?code=00&codeId= HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /api/v1.0/img_codes/4ec89008-5402-4984-9fc6-ea2edeba9889 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:48] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:28:59] "GET /smscode/18979907572?code=151&codeId=7fcd6ac3-8746-446c-86c7-6faf257460f2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:29:07] "GET /api/v1.0/img_codes/3f2cb0dc-1ba6-40fc-9e37-263eb1d56361 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:29:14] "GET /smscode/18979907572?code=wptq&codeId=7fcd6ac3-8746-446c-86c7-6faf257460f2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:29:17] "GET /smscode/18979907572?code=wptq&codeId=7fcd6ac3-8746-446c-86c7-6faf257460f2 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:30:47] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:30:47] "GET /api/v1.0/img_codes/bf83369a-32a4-41f8-b7ef-def780b1fc22 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:31:03] "GET /smscode/18979907572?code=smql&codeId=e65c01a6-f978-48d2-9f58-3d7dbe06efe9 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:31:06] "GET /smscode/18979907572?code=smql&codeId=e65c01a6-f978-48d2-9f58-3d7dbe06efe9 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:31:34] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:31:34] "GET /api/v1.0/img_codes/90ed40f7-56e8-4f0b-88e3-3305db0e4a9c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:31:46] "GET /smscode/18979907572?code=nxik&codeId=2002e3bf-5a23-45d4-9914-2324ed9c7632 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:31:48] "GET /smscode/18979907572?code=nxik&codeId=2002e3bf-5a23-45d4-9914-2324ed9c7632 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:33:15] "GET /smscode/18979907572?code=nxik&codeId=2002e3bf-5a23-45d4-9914-2324ed9c7632 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:34:48] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:34:48] "GET /api/v1.0/img_codes/621b6ff2-df31-48fb-a52a-9a17efeea624 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:34:51] "GET /api/v1.0/img_codes/621b6ff2-df31-48fb-a52a-9a17efeea624 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:35:12] "GET /smscode/18979907572?code=l4tf&codeId=408f3ceb-5837-4c72-9a62-94dc96c6db9d HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:39:45] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:39:46] "GET /api/v1.0/img_codes/b68adc7b-6a16-4d62-bba3-e1a70744d02f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:39:56] "GET /smscode/18979907572?code=njnjnj&codeId=7fbe23ce-3f97-42dc-89d1-a6a1984a4be2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:40:01] "GET /smscode/18979907572?code=njnjnj&codeId=7fbe23ce-3f97-42dc-89d1-a6a1984a4be2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:47:36] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:47:36] "GET /api/v1.0/img_codes/19796bd5-7979-4789-85d6-76bba1bb5d9f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:47:51] "GET /smscode/18979907572?code=4rgs&codeId=248da11e-dba0-4843-b86d-c8fa53972ec9 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:48:00] "GET /smscode/18979907572?code=4rgs&codeId=248da11e-dba0-4843-b86d-c8fa53972ec9 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:48:06] "GET /api/v1.0/img_codes/19796bd5-7979-4789-85d6-76bba1bb5d9f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:48:13] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:48:13] "GET /api/v1.0/img_codes/9fb5f354-5984-41da-9ee7-8c86feb324e6 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:48:32] "GET /smscode/18979907572?code=hhjhj&codeId=9edc1e60-abd3-4dfc-a9a0-f1f8b18760e8 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:33] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:33] "GET /api/v1.0/img_codes/32b3487e-7563-4ff9-b327-bbae344b0142 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET / HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:35] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /api/v1.0/img_codes/ce838d03-fdef-42f7-82d2-0002d520115f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:49:52] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
ERROR verify.py:56 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:50:25] "GET /smscode/18979907572?code=gfd9&codeId=ce838d03-fdef-42f7-82d2-0002d520115f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:51:13] "GET /smscode/18979907572?code=gfd9&codeId=ce838d03-fdef-42f7-82d2-0002d520115f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:51:21] "GET /api/v1.0/img_codes/ce838d03-fdef-42f7-82d2-0002d520115f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:52:04] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:52:04] "GET /api/v1.0/img_codes/5c5de586-5a93-4c24-9f98-4cbd51dcf0cb HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:52:15] "GET /smscode/18979907572?code=hkkl&codeId=5c5de586-5a93-4c24-9f98-4cbd51dcf0cb HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:52:24] "GET /api/v1.0/img_codes/0169e4e4-f69e-4cc9-9091-5e83908a4fa2 HTTP/1.1" 200 -
ERROR verify.py:56 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:52:52] "GET /smscode/18979907572?code=dbsc&codeId=0169e4e4-f69e-4cc9-9091-5e83908a4fa2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:52:52] "GET /smscode/18979907572?code=dbsc&codeId=0169e4e4-f69e-4cc9-9091-5e83908a4fa2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:55:10] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:55:11] "GET /api/v1.0/img_codes/919f5545-2641-4276-b85e-6f24ab8f8abb HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 00:55:42] "GET /smscode/18979907572?code=hqtr&codeId=919f5545-2641-4276-b85e-6f24ab8f8abb HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/libs/yuntongxun/sms.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:57:55] "GET /index.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:57:55] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:57:58] "GET /api/v1.0/image_codes/01ce8461-b540-4d23-af40-b075605c4412 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:04] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:04] "GET /api/v1.0/image_codes/2fbed7be-fcf4-45c2-99ec-f3de1ff0c5a3 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:04] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:05] "GET /api/v1.0/image_codes/15c8d927-339d-4efc-b9cd-129400b5022a HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:06] "GET /api/v1.0/image_codes/6eee9511-73dd-435a-b638-1e80370b23c0 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:07] "GET /api/v1.0/image_codes/a7e46444-5c8e-4a7d-8ba7-cf15be2c1e79 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:07] "GET /api/v1.0/image_codes/6a441d93-f9aa-41f8-80db-905b7f71a9da HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 16:58:09] "GET /api/v1.0/image_codes/6a441d93-f9aa-41f8-80db-905b7f71a9da HTTP/1.1" 404 -
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:33] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:34] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:34] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:34] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:34] "GET /api/v1.0/img_codes/671937e6-a824-48c5-8d46-ee34c55d5604 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:00:34] "GET /favicon.ico HTTP/1.1" 200 -
ERROR verify.py:58 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:01:13] "GET /smscode/18979907572?code=uv3h&codeId=671937e6-a824-48c5-8d46-ee34c55d5604 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:02:48] "GET /smscode/18979907572?code=uv3&codeId=671937e6-a824-48c5-8d46-ee34c55d5604 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:04:37] "GET /api/v1.0/img_codes/573594e1-1097-4b8b-adfd-5b6f9594978f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:04:41] "GET /smscode/18979907572?code=xsdd&codeId=573594e1-1097-4b8b-adfd-5b6f9594978f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:16:52] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:16:52] "GET /api/v1.0/img_codes/00f7dda9-fa4b-4c55-9105-5a1802017e80 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:16:52] "GET /favicon.ico HTTP/1.1" 200 -
ERROR verify.py:58 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:17:22] "GET /smscode/18979907572?code=zspf&codeId=00f7dda9-fa4b-4c55-9105-5a1802017e80 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:17:22] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:20:57] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:20:57] "GET /api/v1.0/img_codes/c2e6007d-76a6-46b5-8160-103ee0789889 HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:20:58] "GET /api/v1.0/img_codes/ddc3185b-4d11-4b62-9fe1-3d5806ccf402 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:20:58] "GET /api/v1.0/img_codes/c2e6007d-76a6-46b5-8160-103ee0789889 HTTP/1.1" 200 -
ERROR verify.py:58 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:21:35] "GET /smscode/18979907572?code=8ojb&codeId=ddc3185b-4d11-4b62-9fe1-3d5806ccf402 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:46:14] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:46:15] "GET /api/v1.0/img_codes/6ee5a62c-1d24-4b8a-9b4b-02d6435ffdd6 HTTP/1.1" 200 -
ERROR verify.py:58 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:46:46] "GET /smscode/18979907572?code=jprf&codeId=6ee5a62c-1d24-4b8a-9b4b-02d6435ffdd6 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:47:24] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:47:25] "GET /api/v1.0/img_codes/7fce0a67-2052-480e-92d7-4ef4f666dc62 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /api/v1.0/img_codes/7b0c6f82-1204-4a4d-8948-0bc0d5f4526a HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /api/v1.0/img_codes/ed540921-8852-4ff5-beda-0d579993856e HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /api/v1.0/img_codes/b4f40f42-3708-47cf-a816-29d74e3525d5 HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /api/v1.0/img_codes/fbe3f8e5-5d4e-44d6-bb6b-a84d98e54cfa HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /api/v1.0/img_codes/e56bc0d3-b1dc-4559-80e4-b8fd92ea9985 HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /api/v1.0/img_codes/0840ca13-b757-4ae7-928c-74cee8a79c61 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:41] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:42] "GET /api/v1.0/img_codes/e1224713-45d4-4969-b2e9-b894267ae50c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:52:42] "GET /favicon.ico HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:53:15] "GET /smscode/18979907572?code=qhyd&codeId=e1224713-45d4-4969-b2e9-b894267ae50c HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:54:21] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:54:21] "GET /api/v1.0/img_codes/60094446-8598-4f37-8c15-a514290b6207 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:55:15] "GET /smscode/18979907572?code=db7b&codeId=60094446-8598-4f37-8c15-a514290b6207 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:55:15] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:55:15] "GET /api/v1.0/img_codes/e694f1b3-203d-425c-a71a-a762ff7b43ed HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:55:47] "GET /smscode/18979907572?code=hncf&codeId=e694f1b3-203d-425c-a71a-a762ff7b43ed HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:56:07] "GET /smscode/18979907572?code=hncf&codeId=e694f1b3-203d-425c-a71a-a762ff7b43ed HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:56:09] "GET /api/v1.0/img_codes/9036e29c-9f1f-4778-9814-e046bf8aaa7f HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:21] "GET /smscode/18979907572?code=5gz4&codeId=9036e29c-9f1f-4778-9814-e046bf8aaa7f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:22] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:22] "GET /api/v1.0/img_codes/d49ec040-2095-4e36-94e6-3f6c69839125 HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:47] "GET /smscode/18979907572?code=6xy5&codeId=d49ec040-2095-4e36-94e6-3f6c69839125 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:51] "GET /smscode/18979907572?code=6xy5&codeId=d49ec040-2095-4e36-94e6-3f6c69839125 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:52] "GET /smscode/18979907572?code=6xy5&codeId=d49ec040-2095-4e36-94e6-3f6c69839125 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:58] "GET /api/v1.0/img_codes/89b3681d-6a39-4dbf-a659-e45eaecdfb58 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 17:58:59] "GET /api/v1.0/img_codes/70536c1a-9830-44a2-817f-6af071d79a6a HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:00:31] "GET /smscode/18979907572?code=ullx&codeId=70536c1a-9830-44a2-817f-6af071d79a6a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:00:59] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:00:59] "GET /api/v1.0/img_codes/cbb2a40c-867c-4e96-829c-7082d1eebee1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:00:59] "GET /favicon.ico HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:10] "GET /smscode/18979907572?code=jbpx&codeId=cbb2a40c-867c-4e96-829c-7082d1eebee1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:16] "GET /smscode/18979907572?code=jbpx&codeId=cbb2a40c-867c-4e96-829c-7082d1eebee1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:17] "GET /smscode/18979907572?code=jbpx&codeId=cbb2a40c-867c-4e96-829c-7082d1eebee1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:23] "GET /smscode/18979907572?code=jbpx&codeId=cbb2a40c-867c-4e96-829c-7082d1eebee1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:24] "GET /smscode/18979907572?code=jbpx&codeId=cbb2a40c-867c-4e96-829c-7082d1eebee1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:27] "GET /api/v1.0/img_codes/8fb99b58-5c5f-4b14-86a3-b1e2fea7743c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:28] "GET /api/v1.0/img_codes/52368f56-514d-4f99-9732-67ebbabdbc15 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:30] "GET /api/v1.0/img_codes/59f2f7a2-0b81-44da-b101-136d4f8025a5 HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:37] "GET /smscode/18979907572?code=7bnf&codeId=59f2f7a2-0b81-44da-b101-136d4f8025a5 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:40] "GET /api/v1.0/img_codes/c75f106c-ad10-4da7-bba8-a93a2715e50f HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:48] "GET /smscode/18979907572?code=ljoe&codeId=c75f106c-ad10-4da7-bba8-a93a2715e50f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:54] "GET /smscode/18979907572?code=ljoe&codeId=c75f106c-ad10-4da7-bba8-a93a2715e50f HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:01:57] "GET /api/v1.0/img_codes/cbb1e872-b721-48ab-b3ac-93f5d9202f6a HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:02:03] "GET /smscode/18979907572?code=otyl&codeId=cbb1e872-b721-48ab-b3ac-93f5d9202f6a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:02:14] "GET /api/v1.0/img_codes/945fc949-e79b-4d3b-899b-3ceab06d9312 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:02:26] "GET /smscode/18979907572?code=3aut&codeId=945fc949-e79b-4d3b-899b-3ceab06d9312 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:02:28] "GET /api/v1.0/img_codes/985fd0be-942a-4d34-926d-e30cf80c8a49 HTTP/1.1" 200 -
ERROR verify.py:57 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 18:02:55] "GET /smscode/18979907572?code=bysj&codeId=985fd0be-942a-4d34-926d-e30cf80c8a49 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:45:54] "GET /smscode/18979907572?code=bysj&codeId=985fd0be-942a-4d34-926d-e30cf80c8a49 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:46:16] "GET /smscode/18979907572?code=bysj&codeId=985fd0be-942a-4d34-926d-e30cf80c8a49 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:46:32] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:46:41] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:46:42] "GET /api/v1.0/img_codes/02aee9e8-f6ef-4413-841c-b73a7e86b73c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:46:42] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:46:53] "GET /smscode/18979907572?code=yaxp&codeId=02aee9e8-f6ef-4413-841c-b73a7e86b73c HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:47:57] "GET /smscode/18979907572?code=yaxp&codeId=02aee9e8-f6ef-4413-841c-b73a7e86b73c HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:49:30] "GET /api/v1.0/img_codes/4ff196b2-34fe-4790-b10f-366f3b65c56a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:49:31] "GET /api/v1.0/img_codes/a7653e27-8ae4-476a-a693-40645101f9bf HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:49:44] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:49:45] "GET /api/v1.0/img_codes/5b3cb67d-b23c-4780-88aa-fef2a5186b8a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:49:56] "GET /smscode/18979907572?code=xeav&codeId=5b3cb67d-b23c-4780-88aa-fef2a5186b8a HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:10] "GET /smscode/18979907572?code=xeav&codeId=5b3cb67d-b23c-4780-88aa-fef2a5186b8a HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:28] "GET /api/v1.0/img_codes/e117a130-fe62-41bb-97a3-fdd166bef910 HTTP/1.1" 200 -
ERROR verify.py:60 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:50:59] "GET /api/v1.0/sms_codes/18979907572?code=wyqa&codeId=e117a130-fe62-41bb-97a3-fdd166bef910 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:00] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:00] "GET /api/v1.0/img_codes/48b39b7f-ba07-493b-b67e-c358f27c9154 HTTP/1.1" 200 -
ERROR verify.py:60 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:15] "GET /api/v1.0/sms_codes/18979907572?code=lmkv&codeId=48b39b7f-ba07-493b-b67e-c358f27c9154 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:20] "GET /api/v1.0/img_codes/6c431566-934b-4508-8f0a-23fc309f96e1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:21] "GET /api/v1.0/sms_codes/18979907572?code=lmkv&codeId=6c431566-934b-4508-8f0a-23fc309f96e1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:28] "GET /api/v1.0/sms_codes/18979907572?code=zoxg&codeId=6c431566-934b-4508-8f0a-23fc309f96e1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:31] "GET /api/v1.0/img_codes/325562f7-354a-46b4-ae1a-4d1f7f5d1b49 HTTP/1.1" 200 -
ERROR verify.py:60 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:37] "GET /api/v1.0/sms_codes/18979907572?code=qqyl&codeId=325562f7-354a-46b4-ae1a-4d1f7f5d1b49 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:39] "GET /api/v1.0/sms_codes/18979907572?code=qqyl&codeId=325562f7-354a-46b4-ae1a-4d1f7f5d1b49 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:43] "GET /api/v1.0/img_codes/063d1c29-7344-46d3-b0cc-2e04cc96e304 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:44] "GET /api/v1.0/img_codes/9c1fe961-4e4c-4f25-847b-eaf9db7161cf HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:51:58] "GET /api/v1.0/sms_codes/18979907572?code=kexo&codeId=9c1fe961-4e4c-4f25-847b-eaf9db7161cf HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:52:00] "GET /api/v1.0/img_codes/a362509e-9a4a-4cb9-9bef-0fc9cf7b83f4 HTTP/1.1" 200 -
ERROR verify.py:60 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [13/Feb/2018 20:52:06] "GET /api/v1.0/sms_codes/18979907572?code=zjl6&codeId=a362509e-9a4a-4cb9-9bef-0fc9cf7b83f4 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/css/ihome/main.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/css/ihome/register.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/js/jquery.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/css/reset.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/images/logo@128x59.png HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/js/ihome/register.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:15] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 10:43:16] "GET /api/v1.0/img_codes/64b13f4d-187a-41b2-a7c4-74cb98feaa7a HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:01:38] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:01:38] "GET /api/v1.0/img_codes/5edbd8da-f242-49a3-bc4e-61ca730c3732 HTTP/1.1" 200 -
ERROR verify.py:65 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:03:02] "GET /api/v1.0/sms_codes/18979907572?code=nyfj&codeId=5edbd8da-f242-49a3-bc4e-61ca730c3732 HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:03:06] "GET /api/v1.0/sms_codes/18979907572?code=nyfj&codeId=5edbd8da-f242-49a3-bc4e-61ca730c3732 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:05:19] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:05:19] "GET /api/v1.0/img_codes/be1c9e41-ae6a-4ac8-9816-80f618a0b79f HTTP/1.1" 200 -
ERROR verify.py:65 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:05:32] "GET /api/v1.0/sms_codes/18979907572?code=kgqp&codeId=be1c9e41-ae6a-4ac8-9816-80f618a0b79f HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:07:36] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:07:36] "GET /api/v1.0/img_codes/e8cfdf45-184c-4daa-8e6e-4eca077104f9 HTTP/1.1" 200 -
ERROR verify.py:65 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:07:56] "GET /api/v1.0/sms_codes/18979907572?code=ypd4&codeId=e8cfdf45-184c-4daa-8e6e-4eca077104f9 HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:08:00] "GET /api/v1.0/sms_codes/18979907572?code=ypd4&codeId=e8cfdf45-184c-4daa-8e6e-4eca077104f9 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:08:05] "GET /api/v1.0/sms_codes/18979907572?code=ypd4&codeId=e8cfdf45-184c-4daa-8e6e-4eca077104f9 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:08:47] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:08:47] "GET /api/v1.0/img_codes/5b667a41-2166-415a-afc1-e134006ace27 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:08:59] "GET /api/v1.0/sms_codes/18979907572?code=w58h&codeId=5b667a41-2166-415a-afc1-e134006ace27 HTTP/1.1" 500 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:20] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:20] "GET /api/v1.0/img_codes/638b52b2-701e-4ca8-bf71-3f9ecd332829 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:22] "GET /api/v1.0/img_codes/5a9a7e0b-107a-4443-8c4d-92b63456e23c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:32] "GET /api/v1.0/sms_codes/18979907572?code=p72c&codeId=5a9a7e0b-107a-4443-8c4d-92b63456e23c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:34] "GET /api/v1.0/img_codes/77d738f9-f51a-4254-b6ed-e7a2d396d209 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:35] "GET /api/v1.0/img_codes/0d7b2586-cdc2-408e-bed4-0f9a55e74a42 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:42] "GET /api/v1.0/sms_codes/18979907572?code=gb34&codeId=0d7b2586-cdc2-408e-bed4-0f9a55e74a42 HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:10:46] "GET /api/v1.0/sms_codes/18979907572?code=gb34&codeId=0d7b2586-cdc2-408e-bed4-0f9a55e74a42 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:12:43] "GET /api/v1.0/sms_codes/18979907572?code=gb34&codeId=0d7b2586-cdc2-408e-bed4-0f9a55e74a42 HTTP/1.1" 200 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:12:43] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:12:43] "GET /api/v1.0/img_codes/8f7c48d8-b27f-4081-b1bf-6c7b87a8c543 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:12:43] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:13:58] "GET /api/v1.0/sms_codes/18979907572?code=hegm&codeId=8f7c48d8-b27f-4081-b1bf-6c7b87a8c543 HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:14:03] "GET /api/v1.0/sms_codes/18979907572?code=hegm&codeId=8f7c48d8-b27f-4081-b1bf-6c7b87a8c543 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:14:57] "GET /api/v1.0/img_codes/8765c57b-092e-4028-bee3-9e0312ee1dd4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:15:12] "GET /api/v1.0/sms_codes/18979907572?code=hegm&codeId=8f7c48d8-b27f-4081-b1bf-6c7b87a8c543 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:15:18] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:15:18] "GET /api/v1.0/img_codes/d2c54d61-e923-4c76-8b36-7013605ff3d4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:16:03] "GET /api/v1.0/sms_codes/18979907572?code=hj&codeId=d2c54d61-e923-4c76-8b36-7013605ff3d4 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:16:08] "GET /api/v1.0/img_codes/5917201c-7eb7-48d8-bbc5-266808e08564 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:16:16] "GET /api/v1.0/sms_codes/18979907572?code=vsrq&codeId=5917201c-7eb7-48d8-bbc5-266808e08564 HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:16:38] "GET /api/v1.0/sms_codes/18979907572?code=vsrq&codeId=5917201c-7eb7-48d8-bbc5-266808e08564 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:20:52] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:20:52] "GET /api/v1.0/img_codes/67549a8e-2d9e-4691-bea1-d6880acae890 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:21:36] "GET /api/v1.0/sms_codes/18979907572?code=zvas&codeId=67549a8e-2d9e-4691-bea1-d6880acae890 HTTP/1.1" 500 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:25:23] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:25:23] "GET /api/v1.0/img_codes/10979819-6884-41e5-8645-13784542fb3c HTTP/1.1" 200 -
ERROR verify.py:69 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:26:06] "GET /api/v1.0/sms_codes/18979907572?code=xrzq&codeId=10979819-6884-41e5-8645-13784542fb3c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:22] "GET /api/v1.0/sms_codes/18979907572?code=xrzq&codeId=10979819-6884-41e5-8645-13784542fb3c HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:26] "GET /api/v1.0/img_codes/bedadc7b-4176-4377-b2c2-dcea7fed9f83 HTTP/1.1" 200 -
ERROR verify.py:69 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:32] "GET /api/v1.0/sms_codes/18979907572?code=rtkt&codeId=bedadc7b-4176-4377-b2c2-dcea7fed9f83 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:35] "GET /api/v1.0/img_codes/88314a58-dc70-4bcc-8688-cf1bf4e67a1a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:37] "GET /api/v1.0/img_codes/facd0b34-dcea-4bc3-a2fe-dd817b5421cb HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:38] "GET /api/v1.0/img_codes/87441f9f-eaa8-4196-a567-ea64b18d55bd HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:27:39] "GET /api/v1.0/img_codes/44ca870e-c933-44dc-a2a9-13ba781f837b HTTP/1.1" 200 -
ERROR verify.py:69 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:28:18] "GET /api/v1.0/sms_codes/18979907572?code=oxru&codeId=44ca870e-c933-44dc-a2a9-13ba781f837b HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:46:33] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:46:33] "GET /api/v1.0/img_codes/063aebb2-f587-4e5b-a828-5aa40563b371 HTTP/1.1" 200 -
ERROR verify.py:66 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [14/Feb/2018 11:47:14] "GET /api/v1.0/sms_codes/18979907572?code=4x8l&codeId=063aebb2-f587-4e5b-a828-5aa40563b371 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/config.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:09] "GET /register.html HTTP/1.1" 304 -
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 202, in execute
    write(b'')
  File "/home/python/.virtualenvs/flask_py2/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:10] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/css/reset.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/css/ihome/main.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/css/ihome/register.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/js/jquery.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/images/logo@128x59.png HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:13] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:17] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:27:21] "GET /api/v1.0/img_codes/84ee2697-8fc6-4c7a-baa1-c83e8933a94e HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907571', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:28:28] "GET /api/v1.0/sms_codes/18979907571?code=Mcf7&codeId=84ee2697-8fc6-4c7a-baa1-c83e8933a94e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:28:56] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:04] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:38] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:38] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:38] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:38] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:38] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:39] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:29:51] "GET /api/v1.0/user/ HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:28] "GET /api/v1.0/img_codes/d502202f-224e-402b-ab7b-71cf0b8ac131 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907579', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:30:44] "GET /api/v1.0/sms_codes/18979907579?code=ijf8&codeId=d502202f-224e-402b-ab7b-71cf0b8ac131 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:08] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:16] "GET /api/v1.0/user/ HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:16] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:17] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:17] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:17] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:17] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:44] "GET /api/v1.0/user/ HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:44] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:44] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:44] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:31:44] "GET /api/v1.0/user/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:26] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:28] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:35] "GET /api/v1.0/user/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:38] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:40] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:41] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:41] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:41] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:43] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:33:43] "GET /api/v1.0/img_codes/ad02dd98-959c-46b4-9d0c-8d8203dd8299 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907570', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:34:22] "GET /api/v1.0/sms_codes/18979907570?code=bohj&codeId=ad02dd98-959c-46b4-9d0c-8d8203dd8299 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:34:43] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:44:50] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:45:02] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:45:50] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:47:06] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:50:27] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:50:27] "GET /api/v1.0/img_codes/d6361df9-900a-425f-bcff-cc7408d8f243 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:50:44] "GET /api/v1.0/img_codes/02ad915c-7411-4ed7-8ae0-7acadd9fb9b1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:50:45] "GET /api/v1.0/img_codes/080abad9-4e0f-4424-b596-91e9cf4e9522 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:51:20] "GET /api/v1.0/sms_codes/18979907572?code=njbq&codeId=080abad9-4e0f-4424-b596-91e9cf4e9522 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:51:54] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:51] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:51] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:51] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:51] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:51] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:51] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:52] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:52] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:52] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:52] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:52] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:52:52] "GET /api/v1.0/img_codes/f7418b86-1f9c-4faf-84dc-ca8d671a3898 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:53:00] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:55:58] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:01] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:02] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:02] "GET /api/v1.0/img_codes/77966b71-ef0a-486b-990c-27029989bcab HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:21] "GET /api/v1.0/sms_codes/18979907572?code=fvaj&codeId=77966b71-ef0a-486b-990c-27029989bcab HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:24] "GET /api/v1.0/img_codes/21b38489-b011-4feb-a3d8-b3276f47b8a7 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:25] "GET /api/v1.0/img_codes/62c76fe5-678e-420f-b129-3733deef8264 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:32] "GET /api/v1.0/sms_codes/18979907572?code=kjaf&codeId=62c76fe5-678e-420f-b129-3733deef8264 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:56:51] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 00:58:03] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:00:58] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:11] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 01:01:12] "GET /api/v1.0/img_codes/08e88cba-3b6c-4bb4-bf91-f7f07cc4b11e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/css/reset.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/css/ihome/register.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/js/jquery.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/css/ihome/main.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/js/ihome/register.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/images/logo@128x59.png HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:46] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:47] "GET /api/v1.0/img_codes/8e9146fb-56aa-484e-99b7-f7b1261d69ca HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:17:47] "GET /favicon.ico HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:18:28] "GET /api/v1.0/sms_codes/18979907572?code=lmgz&codeId=8e9146fb-56aa-484e-99b7-f7b1261d69ca HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:18:49] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:18:57] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:25:31] "GET /api/v1.0/user/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:26:27] "GET /api/v1.0/user/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:30:28] "GET /api/v1.0/user/ HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:45:57] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:46:10] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:05] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:21] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:22] "GET /api/v1.0/image_codes/48fef49d-29a7-423b-8a9d-25cc92dd2885 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:27] "GET /api/v1.0/image_codes/a6551d35-3304-4597-82e8-292e4003e9f2 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:27] "GET /api/v1.0/image_codes/d29155b9-1397-4ae8-90b1-4b0fe9992fe2 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:28] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:29] "GET /api/v1.0/image_codes/e9a3c1c6-10ba-454e-9988-42a3c6525de8 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:30] "GET /api/v1.0/image_codes/b3c838a1-f0ba-47bb-98ec-f754d76f6d8c HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:30] "GET /api/v1.0/image_codes/61a369d8-579c-4496-82c9-b9687d2e609c HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:30] "GET /api/v1.0/image_codes/141d67c5-6845-4676-931e-3ee11b5d3242 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:48:31] "GET /api/v1.0/image_codes/8c439192-b633-481e-947c-0d5a95dc9a9d HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:49:44] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:49:45] "GET /api/v1.0/image_codes/06496a97-16a9-4af5-9db8-786da01f0876 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:49:57] "GET /api/v1.0/sms_codes/1897907572?image_code=mb3q&image_code_id=06496a97-16a9-4af5-9db8-786da01f0876 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 21:50:39] "GET /api/v1.0/sms_codes/1897907572?image_code=mb3q&image_code_id=06496a97-16a9-4af5-9db8-786da01f0876 HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/verify.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:18] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:18] "GET /api/v1.0/image_codes/42ef7aec-57d3-48de-a58c-50414b8a961f HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:22] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:22] "GET /api/v1.0/image_codes/00476a8e-1ca6-443a-b2cf-71053870d579 HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /api/v1.0/img_codes/0a092116-21a6-4ea1-82c1-38c14cc49313 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:20:38] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:21:05] "GET /api/v1.0/sms_codes/18979907572?code=wwj9&codeId=0a092116-21a6-4ea1-82c1-38c14cc49313 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:21:32] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:21:35] "GET /api/v1.0/user/ HTTP/1.1" 404 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:25:24] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:25:25] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:25:25] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [21/Feb/2018 22:25:26] "POST /api/v1.0/user/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:20] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:20] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:20] "GET /static/css/ihome/main.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:20] "GET /static/css/reset.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:20] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/css/ihome/register.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/js/jquery.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/images/logo@128x59.png HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:23] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:15:24] "GET /api/v1.0/img_codes/56330548-3157-4bbc-aecd-aa82e2c19e06 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:24] "GET /api/v1.0/img_codes/d8f337b7-e66e-4277-8f11-08ecd33b1d34 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:35] "GET /api/v1.0/sms_codes/18979907572?code=qwf3&codeId=d8f337b7-e66e-4277-8f11-08ecd33b1d34 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:56] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:56] "GET /index.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:16:57] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:28:36] "GET /index.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:28:37] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:29:00] "GET /api/v1.0/img_codes/13dc4f11-c5db-44cc-abf2-f94097ca9440 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:29:02] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:29:03] "GET /api/v1.0/img_codes/2cbc0959-bab4-4625-bcaf-739addefdbb1 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:29:17] "GET /api/v1.0/sms_codes/18979907572?code=iumt&codeId=2cbc0959-bab4-4625-bcaf-739addefdbb1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:01] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:02] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:02] "GET /api/v1.0/img_codes/6eb986c9-0c55-4122-b821-2adffcc2ab7e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:02] "GET /favicon.ico HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:25] "GET /api/v1.0/img_codes/714ca31e-2ee9-41c8-94af-0661ee70868d HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907576', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:33] "GET /api/v1.0/sms_codes/18979907576?code=gr8d&codeId=714ca31e-2ee9-41c8-94af-0661ee70868d HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:54] "POST /api/v1.0/users/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:30:57] "POST /api/v1.0/users/ HTTP/1.1" 405 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:01] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:01] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:01] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:01] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:02] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:02] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:20] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:20] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:20] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:21] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:21] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:21] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:21] "GET /api/v1.0/img_codes/1efceccf-da3d-4435-bda2-0eb7ffebc2f2 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:27] "GET /api/v1.0/img_codes/f845b236-485b-4a6f-9000-f95331c2cb8a HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907578', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:33] "GET /api/v1.0/sms_codes/18979907578?code=r8d7&codeId=f845b236-485b-4a6f-9000-f95331c2cb8a HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /index.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:31:53] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:32:12] "GET /api/v1.0/img_codes/18db0bad-f529-476c-8ebc-0f1126f05d7e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:32:14] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:32:14] "GET /api/v1.0/img_codes/c2ee6bb0-ae07-4e63-b954-663a8224672e HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907579', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:32:37] "GET /api/v1.0/sms_codes/18979907579?code=lvea&codeId=c2ee6bb0-ae07-4e63-b954-663a8224672e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:00] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:00] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:43] "GET /api/v1.0/img_codes/15df7695-26be-478f-8163-1a74611f9823 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:33:57] "GET /api/v1.0/img_codes/494309bf-97a6-495c-ae7b-163cb5749cde HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:34:11] "GET /api/v1.0/sms_codes/18979907572?code=htax&codeId=494309bf-97a6-495c-ae7b-163cb5749cde HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:34:30] "POST /api/v1.0/users HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:34:32] "POST /api/v1.0/users HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:34:53] "POST /api/v1.0/users HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:05] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:06] "GET /api/v1.0/img_codes/94c6917e-a0e7-49b9-a0c5-d1924bd04198 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:22] "GET /api/v1.0/img_codes/772fadd3-0112-4a6d-b9eb-bcfb30011811 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:23] "GET /api/v1.0/img_codes/13036b6e-2bc4-456e-81fa-a3be93685029 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:30] "GET /api/v1.0/sms_codes/18979907571?code=ha6t&codeId=13036b6e-2bc4-456e-81fa-a3be93685029 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:32] "GET /api/v1.0/img_codes/6d378e7c-09ae-4445-b480-a35c0881925e HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907571', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:35:39] "GET /api/v1.0/sms_codes/18979907571?code=suyq&codeId=6d378e7c-09ae-4445-b480-a35c0881925e HTTP/1.1" 200 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:36:06] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:36:07] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:36:19] "GET /api/v1.0/users HTTP/1.1" 404 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:36:25] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:38] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:38] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:38] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /api/v1.0/img_codes/e85678ae-9477-456f-857b-06b12c392186 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:39] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907576', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:37:52] "GET /api/v1.0/sms_codes/18979907576?code=w753&codeId=e85678ae-9477-456f-857b-06b12c392186 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /index.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:14] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:15] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:24] "GET /api/v1.0/img_codes/1844d6c9-6a93-4c52-b372-683d0b60c5d1 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:26] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:26] "GET /api/v1.0/img_codes/c547d638-ddf5-4652-b062-0c63c611828e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:44] "GET /api/v1.0/img_codes/66c02f9a-6c24-4693-a8d9-e3549604d8a8 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:38:55] "GET /api/v1.0/sms_codes/18979907572?code=jfsq&codeId=66c02f9a-6c24-4693-a8d9-e3549604d8a8 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:39:15] "POST /api/v1.0/users HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:39:51] "POST /api/v1.0/users HTTP/1.1" 500 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:40:49] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:40:49] "GET /api/v1.0/img_codes/9946259f-43d6-4b45-8f84-ec26901b9be1 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907574', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:41:13] "GET /api/v1.0/sms_codes/18979907574?code=x8ub&codeId=9946259f-43d6-4b45-8f84-ec26901b9be1 HTTP/1.1" 200 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:41:45] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:41:59] "GET /api/v1.0/users HTTP/1.1" 404 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:05] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:17] "GET /api/v1.0/users HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /api/v1.0/img_codes/1557872f-a84a-42c5-81e4-1a8b0bda130e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:42:55] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:03] "GET /api/v1.0/img_codes/b2074856-ba03-4260-a876-d6b3be6e6399 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:12] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:13] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:13] "GET /api/v1.0/img_codes/d21d7d1d-9885-493e-862b-2b057be3f771 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:13] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:15] "GET /api/v1.0/img_codes/896b05ac-2ea5-43c1-a7d2-0ca4bfb12b68 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:17] "GET /api/v1.0/img_codes/d4f3827b-fee0-489e-a043-b24c8cb51ccc HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:17] "GET /api/v1.0/img_codes/699e96c3-c7b8-471d-bd03-30113a3d33e0 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907579', 1)]
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:24] "GET /api/v1.0/sms_codes/18979907579?code=gmey&codeId=699e96c3-c7b8-471d-bd03-30113a3d33e0 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [22/Feb/2018 13:43:41] "POST /api/v1.0/users/ HTTP/1.1" 405 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/css/reset.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/css/ihome/main.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/css/ihome/register.css HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/js/jquery.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:52] "GET /static/images/logo@128x59.png HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:53] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:53] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:53] "GET /api/v1.0/img_codes/cdac7665-ea60-4bc5-a452-3438622e1dbe HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:55] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:24:55] "GET /api/v1.0/img_codes/f3203733-4513-4541-9faa-48b2783146a5 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:25:13] "GET /api/v1.0/img_codes/19354a4b-bdf2-4d82-b415-5464f1856b35 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:25:14] "GET /api/v1.0/img_codes/c697dcba-6df5-46ac-aab6-5bfb8a85cb57 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:25:36] "GET /api/v1.0/sms_codes/18979907572?code=wgzx&codeId=c697dcba-6df5-46ac-aab6-5bfb8a85cb57 HTTP/1.1" 200 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:25:57] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:29] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:30] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:33] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:35] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:35] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:36] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:36] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:36] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:36] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:37] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO csrf.py:251 The CSRF token is missing.
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:37] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /api/v1.0/img_codes/0f2bedee-58b6-475a-8827-cae39edcef9d HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:40] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907571', 1)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:26:49] "GET /api/v1.0/sms_codes/18979907571?code=bfwq&codeId=0f2bedee-58b6-475a-8827-cae39edcef9d HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /index.html HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/css/ihome/index.css HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/js/ihome/index.js HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/images/home01.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/images/home02.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:07] "GET /static/images/home03.jpg HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:27:08] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.1" 404 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:28:25] "GET /api/v1.0/img_codes/91350a01-bc81-4163-ac9e-b9f0d6e0a548 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:29:16] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:29:25] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 09:29:30] "GET /api/v1.0/sms_codes/18979907572?code=bfwq&codeId=91350a01-bc81-4163-ac9e-b9f0d6e0a548 HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/models.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/__init__.py', reloading
INFO _internal.py:87  * Restarting with stat
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:13:42] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:13:44] "GET /api/v1.0/img_codes/8b006950-f941-4999-b3ca-a4c17ce1a37b HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:14:17] "GET /api/v1.0/sms_codes/18979907572?code=4vqx&codeId=8b006950-f941-4999-b3ca-a4c17ce1a37b HTTP/1.1" 200 -
ERROR passport.py:56 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2018, 2, 24, 12, 14, 37, 280596), datetime.datetime(2018, 2, 24, 12, 14, 37, 280625), '18979907572', 'pbkdf2:sha256:50000$uSdY3W6o$952e29d362084153010b13b1cb5007d7590a180daf7348997dc33311ac91314a', '18979907572', None, None, None)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:14:37] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:14:45] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87  * Detected change in '/home/python/flask0000/flask_tryagain/git05/flaskapp/04/ihome/api_1_0/passport.py', reloading
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:18:11] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:18:11] "GET /api/v1.0/img_codes/aae3f807-5734-486e-8eb0-8960d460dc35 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:18:22] "GET /api/v1.0/img_codes/13252792-a292-4ffa-b6ba-4a27ccf1fc27 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:18:23] "GET /api/v1.0/img_codes/00131824-cd21-4319-82d1-ecf0e963bf95 HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:18:25] "GET /api/v1.0/img_codes/7f68b0ae-7abd-456f-8524-56f7830beed2 HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907571', 1)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:18:35] "GET /api/v1.0/sms_codes/18979907571?code=8xyj&codeId=7f68b0ae-7abd-456f-8524-56f7830beed2 HTTP/1.1" 200 -
ERROR passport.py:56 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2018, 2, 24, 12, 19, 1, 241293), datetime.datetime(2018, 2, 24, 12, 19, 1, 241326), '18979907571', 'pbkdf2:sha256:50000$cKLmKEaK$c24c975ad5ec02224435dc09c1c87b12591f2b077661e9d718519ae052182cb3', '18979907571', None, None, None)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 12:19:01] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 16:40:06] "GET /api/v1.0/img_codes/172b179f-b6a1-4e0b-b3f4-8d9ef42e2bbc HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 16:40:14] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 16:40:14] "GET /api/v1.0/img_codes/7e5e3c43-1d18-4270-9148-e36b90a8552e HTTP/1.1" 200 -
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 16:40:14] "GET /favicon.ico HTTP/1.1" 200 -
ERROR verify.py:63 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'SELECT ih_user_profile.create_time AS ih_user_profile_create_time, ih_user_profile.update_time AS ih_user_profile_update_time, ih_user_profile.id AS ih_user_profile_id, ih_user_profile.name AS ih_user_profile_name, ih_user_profile.password_hash AS ih_user_profile_password_hash, ih_user_profile.mobile AS ih_user_profile_mobile, ih_user_profile.real_name AS ih_user_profile_real_name, ih_user_profile.id_card AS ih_user_profile_id_card, ih_user_profile.avatar_url AS ih_user_profile_avatar_url \nFROM ih_user_profile \nWHERE ih_user_profile.mobile = %s \n LIMIT %s'] [parameters: ('18979907572', 1)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 16:40:57] "GET /api/v1.0/sms_codes/18979907572?code=e3mi&codeId=7e5e3c43-1d18-4270-9148-e36b90a8552e HTTP/1.1" 200 -
ERROR passport.py:56 (_mysql_exceptions.ProgrammingError) (1146, "Table 'db_python04.ih_user_profile' doesn't exist") [SQL: u'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2018, 2, 24, 16, 41, 19, 314574), datetime.datetime(2018, 2, 24, 16, 41, 19, 314597), '18979907572', 'pbkdf2:sha256:50000$6VvRKkhR$2ef7f23fbf54bbe314598e42992e7030f04e13ba4123cdc39ef7712ffe9f9217', '18979907572', None, None, None)]
INFO _internal.py:87 127.0.0.1 - - [24/Feb/2018 16:41:19] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:87  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:87  * Restarting with stat
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:87  * Debugger PIN: 225-739-483
